### DISCUSSION TITLE
How can I set a custom save location?

### DISCUSSION THREAD

**Discussion by ySergi** [lllyasviel/Fooocus#2896] [Q&A] [answered]
By default everything is saved on "/outputs/", I'd like to save the images on another location.

**Comment by ILikeToasters** [comment, marked as answer]
Edit the config.txt file in the Fooocus folder. There are more details in the config_modification_tutorial.txt. Be sure to use the \\ and not just a single \. 
<img width="602" alt="Screenshot 2024-05-10 133230" src="https://github.com/lllyasviel/Fooocus/assets/4357657/dc11af9a-357f-40f2-8a5b-498922d5e734">
<img width="306" alt="Screenshot 2024-05-10 133328" src="https://github.com/lllyasviel/Fooocus/assets/4357657/c0683065-dd0d-40c5-9741-8b7b20208db8">
